core/src/main/java/org/elasticsearch/action/support/replication/TransportReplicationAction.java

```diff
@@ -89,7 +90,7 @@
     final protected ClusterService clusterService;
     final protected IndicesService indicesService;
     final private ShardStateAction shardStateAction;
-    final private WriteConsistencyLevel defaultWriteConsistencyLevel;
+    final private int defaultWaitOnActiveShards;
```

Comment:
indeed, but I think it’s good to do it in a different change?

> On 24 Jun 2016, at 10:08, Yannick Welsch <notifications@github.com> wrote:
> 
> In core/src/main/java/org/elasticsearch/action/support/replication/TransportReplicationAction.java:
> 
> > @@ -89,7 +90,7 @@
> >      final protected ClusterService clusterService;
> >      final protected IndicesService indicesService;
> >      final private ShardStateAction shardStateAction;
> > -    final private WriteConsistencyLevel defaultWriteConsistencyLevel;
> > +    final private int defaultWaitOnActiveShards;
> 
> I think we agreed on making this setting updatable (cluster setting).
> 
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub, or mute the thread.
>